crates/orchestrator/src/routes.rs

```diff
@@ -1,16 +1,10 @@
 use axum::http::StatusCode;
 use axum::response::IntoResponse;
-use axum::routing::{get, post};
+use axum::routing::get;
 use axum::Router;
 
-use crate::controllers::jobs_controller;
-
 pub fn app_router() -> Router {
-    Router::new()
-        .route("/health", get(root))
-        .nest("/v1/dev", dev_routes())
-        .nest("/v1/job", job_routes())
-        .fallback(handler_404)
+    Router::new().route("/health", get(root)).nest("/v1/dev", dev_routes()).fallback(handler_404)
```

Comment:
How is this change related to the PR?

Reply:
This is a valid concern,
While the tests for `Job_controller` were being written it was realised that we are not using it anymore and hence it can be deleted, we agree, ideally this is to be done on a separate PR.

Proposed Solution :
- Have stricter decouples of PR from next time.
- Keep the commit with this PR.
- Mention removal of `job_controller` in the `CHANGELOG`.

Reply:
You can keep it in this PR, but mention it in the changelog and in the PR description.

Reply:
sure